socorro/collector/wsgi_breakpad_collector.py

```diff
@@ -86,6 +91,33 @@ def POST(self, *args):
         else:
             raw_crash.legacy_processing = int(raw_crash.legacy_processing)
 
+        # We want to capture the crash report size, but need to differentiate
+        # between compressed vs. uncompressed data as well as accepted vs.
+        # rejected data.
+        crash_report_size = self._get_content_length()
+        is_compressed = self._is_content_gzipped()
+        is_accepted = (raw_crash.legacy_processing in (ACCEPT, DEFER))
+
+        try:
+            metrics_data = {}
+            size_key = '_'.join([
+                'crash_report_size',
+                'accepted' if is_accepted else 'rejected',
+                'compressed' if is_compressed else 'uncompressed',
+            ])
+            metrics_data = {
+                size_key: crash_report_size
+            }
+            self.metrics.capture_stats(metrics_data)
+        except Exception as exc:
```

Comment:
Perhaps only the `self.metrics.capture_stats(metrics_data)` line should be in the `try:except:`. 
Or perhaps the `self._get_content_length()` should be included too?

However, we might be best to put this into stage for a couple of days/weeks and see if we ever get any exceptions on any of the lines *other* than `self.metrics.capture_stats(metrics_data)`.